README.md

```diff
@@ -57,15 +57,20 @@ These are the dependencies needed for building Wayfire.
 These are the dependencies needed for building wlroots, and should be installed before building it.
 They are relevant for cases when the system doesn't have a version of wlroots installed.
 
-#### DRM Backend (optional, but you most likely want this)
+#### DRM Backend (required)
 
 - [libdisplay-info-dev](https://gitlab.freedesktop.org/emersion/libdisplay-info)
 - [hwdata-dev](https://github.com/vcrhonek/hwdata)
 
-#### Session Provider (optional, recommended)
+#### GLES2 renderer (required)
+- [libglvnd](https://gitlab.freedesktop.org/glvnd/libglvnd)
```

Comment:
Is libglvnd an actual dependency of wlroots? I don't see it listed on their README. Also by looking at the meson.build files, the gles2 renderer needs egl, gbm and glesv2

Reply:
The most direct dependency is libglvnd as it provided the glesv2 pkgconfig file that is used by wlroots. 

https://gitlab.freedesktop.org/glvnd/libglvnd/-/blob/master/src/GLESv2/glesv2.pc.in?ref_type=heads

Reply:
but egl and gbm (so mesa) should also be there.

Reply:
I think we should be even more explicit about these things, because on other distros (like mine, Fedora) the mesa packages is split in multiple parts. For example GBM is not mesa-devel but mesa-libgbm-devel. Maybe just specify that we need the gbm and egl components of it.

Reply:
```suggestion
- [libglvnd](https://gitlab.freedesktop.org/glvnd/libglvnd)
- [mesa](https://gitlab.freedesktop.org/mesa/mesa) (with libEGL and gbm support)
```

Atleast on Fedora I notice that the minimum required explicit packages would be [libglvnd-devel](https://koji.fedoraproject.org/koji/rpminfo?rpmID=37559750) and [mesa-libgbm-devel](https://koji.fedoraproject.org/koji/rpminfo?rpmID=38042568)

Reply:
looks great :)